### PR TITLE
Add Bugsnag Integrity header to Expo delivery

### PR DESCRIPTION
**Pull request by imjoehaines**
## Goal

Adds the Bugsnag-Integrity header to Expo delivery, using `expo-crypto` to calculate the sha1 hash of the payload

### CODE REVIEW

File: packages/delivery-expo/package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
   "license": "MIT",
   "dependencies": {
     "@react-native-community/netinfo": "5.9.6",
-    "expo-file-system": "~9.2"
+    "expo-file-system": "~9.2",
+    "expo-crypto": "~8.3.0"
```
Comment (bengourley) [inline packages/delivery-expo/package.json:22]: How did you wind up with this version?

As of the latest Expo (v40) the latest version of the crypto module is 8.1.0: https://github.com/expo/expo/blob/c326745ae16d7b6bbad86de69a61ea9606085b7a/changelogVersions.json#L27

I think we ought to pin it to that.

**Review by bengourley — APPROVED** [review]
Looks good aside from maybe the exact version.

File: packages/delivery-expo/package.json (hunk 1 above)
Comment (imjoehaines) [reply packages/delivery-expo/package.json:22]: I don't think SDK 40 has been released, has it? This page suggests 39 is the latest version https://docs.expo.io/workflow/upgrading-expo-sdk-walkthrough/

This matches the version constraint for Expo 39: https://github.com/expo/expo/blob/sdk-39/packages/expo/bundledNativeModules.json#L25

Comment (bengourley) [reply packages/delivery-expo/package.json:22]: huh, how confusing, the depedency for the unrelease v40 is _lower_ than that of 39 😕

Ok, as long as it matches v39 I'm happy.

Comment (imjoehaines) [reply packages/delivery-expo/package.json:22]: Just noticed we linked different files — for SDK 39 the version constraint in [`bundledNativeModules` is `~8.3.0`](https://github.com/expo/expo/blob/sdk-39/packages/expo/bundledNativeModules.json#L25) but in [`changelogVersions` it's `8.1.0`](https://github.com/expo/expo/blob/sdk-39/changelogVersions.json#L27)

Expo Crypto has native components (https://github.com/expo/expo/tree/sdk-39/packages/expo-crypto) so I think `bundledNativeModules` is the right one to use (?) I'm not sure why it's in both files & why they are different versions though :s

**Pull request merged**